pkg/golinters/mnd.go

```diff
@@ -0,0 +1,21 @@
+package golinters
+
+import (
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+	"golang.org/x/tools/go/analysis"
+
+	magic_numbers "github.com/tommy-muehle/go-mnd"
+)
+
+func NewMnd() *goanalysis.Linter {
+	analyzers := []*analysis.Analyzer{
+		magic_numbers.Analyzer,
+	}
+
+	return goanalysis.NewLinter(
+		"magicnumber",
+		"checks whether magic number detector is used",
+		analyzers,
+		nil,
+	).WithLoadMode(goanalysis.LoadModeTypesInfo)
```

Comment:
I guess we need only `LoadModeSyntax` mode. I didn't find types usage in the linter.

Reply:
Thanks 💯